lambda_function/aws/resources/acm_cert_handler.py

```diff
@@ -0,0 +1,80 @@
+import json
+import logging
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import boto3
+import consts
+from aws.resources.base_handler import BaseHandler
+from port.entities import create_entities_json, handle_entities
+
+logger = logging.getLogger(__name__)
+
+
+class ACMHandler(BaseHandler):
+    def handle(self):
+        for region in list(self.regions):
+            aws_acm_client = boto3.client("acm", region_name=region)
+            try:
+                paginator = aws_acm_client.get_paginator('list_certificates')
```

Comment:
Add support for [list_parameters](https://github.com/port-labs/port-aws-exporter/blob/2bb2a6d1a4cfc3270df896193840b54d4e939d80/lambda_function/aws/resources/cloudformation_handler.py#L22) to allow to other filters than the default filters